src/Field/Base/PartsField.php

```diff
@@ -150,6 +150,33 @@ final public function addInputContainerAttributes(array $attributes): static
         return $new;
     }
 
+    /**
+     * Replace input container tag CSS classes with a new set of classes.
+     *
+     * @param string|null ...$class One or many CSS classes.
+     */
+    final public function inputContainerClass(?string ...$class): static
+    {
+        $new = clone $this;
+        $new->inputContainerAttributes['class'] = array_filter($class, static fn ($c) => $c !== null);
+        return $new;
+    }
+
+    /**
+     * Add one or more CSS classes to the input container tag.
+     *
+     * @param string|null ...$class One or many CSS classes.
+     */
+    final public function addInputContainerClass(?string ...$class): static
+    {
+        $new = clone $this;
+        Html::addCssClass(
+            $new->inputContainerAttributes,
+            array_filter($class, static fn ($c) => $c !== null),
```

Comment:
Why do we need null check here?

Reply:
Agree. Here not need null values in function parameters. I'm remove this in separate PR (at once in all places).